corehq/apps/app_manager/templates/app_manager/partials/build_errors.html

```diff
@@ -317,5 +317,12 @@
                 {% trans "Errors Building an Application" %}
             </a>
         </p>
+        {% if in_iframe %}
+        <script>
+            $('#build-errors a').each(function(index) {
+                $(this).attr('target', '_parent');
+            });
+        </script>
+        {% endif %}
```

Comment:
I don't follow...as long as the js is included on every page that uses this partial, isn't that still fine? I'm thinking of something like 
```$(document).on("iframe.preview-phone-container a", "click", function() {
   document.location = $(this).attr("href");
});```